docs/Embedded/EnkluVolume.md

```diff
@@ -0,0 +1,18 @@
+---
+id: EnkluVolume
+title: Embedding an Experience
+---
+
+_\[Note: Access to this feature requires a [Studio Subscription](https://enklu.com/pricing). [Contact us](/contact) for details.\]_
+
+To embed a scene in your Unity project, add an EnkluVolume prefab to your scene. Find it by searching for "EnkluVolume" in the project tab and ensuring the "In Packages" option is selected. Once the prefab is in your scene, select it and log in to your [Enklu Cloud account](https://cloud.enklu.com) in the inspector tab.
```

Comment:
I'm wondering if we really want to link to cloud here because I don't feel it really adds much value and I foresee some misunderstandings.  The instruction is to `log in`, but then we link to the cloud log in page which is not where we want them to log in.